src/components/BridgeBanner.tsx

```diff
@@ -0,0 +1,81 @@
+import { Trans } from '@lingui/macro'
+import { useWeb3React } from '@web3-react/core'
+import { getChainInfo, isSupportedChainId } from 'constants/chainInfo'
+import { ArrowUpRight } from 'icons'
+import styled from 'styled-components/macro'
+
+import Column from './Column'
+import ExternalLink from './ExternalLink'
+
+const ContentWrapper = styled.div<{ backgroundColor: string }>`
+  background: ${({ backgroundColor }) => backgroundColor};
+  border-radius: 1.25em;
+  display: flex;
+  flex-direction: row;
+  margin-top: 1em;
+  overflow: hidden;
+  position: relative;
+  width: 100%;
+`
+const BridgeLink = styled(ExternalLink)<{ color: string }>`
+  align-items: center;
+  border-radius: 0.5em;
+  color: ${({ color }) => color};
+  display: flex;
+  font-size: 1em;
+  justify-content: space-between;
+  padding: 0.75em;
+  text-decoration: none !important;
```

Comment:
`!important` should be used as a last resort 🤔 is there another way u can set this property, maybe by making the selector more specific?

Reply:
was leftover from copying from interface, the !important seems to not actually be _imporant_ here so I'm removing 👍

Reply:
sweeet